src/frontend/hooks/useSendInnSkjema.tsx

```diff
@@ -27,28 +28,37 @@ export const useSendInnSkjema = (): {
     const sendInnSkjema = async (): Promise<[boolean, ISøknadKontrakt]> => {
         settInnsendingStatus({ status: RessursStatus.HENTER });
 
-        const formatert: ISøknadKontrakt = dataISøknadKontraktFormat(
-            valgtLocale,
-            søknad,
-            tekster(),
-            tilRestLocaleRecord
-        );
-
-        const res = await sendInn<ISøknadKontrakt>(
-            formatert,
-            axiosRequest,
-            `${soknadApiProxyUrl}/soknad/kontantstotte/v4`,
-            (res: AxiosError) => {
-                const responseData = res.response?.data as Ressurs<IKvittering>;
-                if (responseData && erModellMismatchResponsRessurs(responseData)) {
-                    settSisteModellVersjon(responseData.data.modellVersjon);
+        try {
+            const formatert: ISøknadKontrakt = dataISøknadKontraktFormat(
+                valgtLocale,
+                søknad,
+                tekster(),
+                tilRestLocaleRecord
+            );
+
+            const res = await sendInn<ISøknadKontrakt>(
+                formatert,
+                axiosRequest,
+                `${soknadApiProxyUrl}/soknad/kontantstotte/v4`,
+                (res: AxiosError) => {
+                    const responseData = res.response?.data as Ressurs<IKvittering>;
+                    if (responseData && erModellMismatchResponsRessurs(responseData)) {
+                        settSisteModellVersjon(responseData.data.modellVersjon);
+                    } else {
+                        //Denne skal feile mykt, med en custom feilmelding til brukeren. Kaster dermed ingen feil her.
+                        Sentry.captureException(
+                            new Error('Klarte ikke sende inn søknaden', { cause: res.message })
+                        );
+                    }
                 }
-            }
-        );
+            );
 
-        settInnsendingStatus(res);
+            settInnsendingStatus(res);
 
-        return [res.status === RessursStatus.SUKSESS, formatert];
+            return [res.status === RessursStatus.SUKSESS, formatert];
+        } catch (e) {
+            throw new Error('Søknaden feilet på innsending', { cause: e });
+        }
```

Comment:
Litt nitpick men syns det er greit å unngå variabler som kun inneholder 1 bokstav 

```suggestion
        } catch (error) {
            throw new Error('Søknaden feilet på innsending', { cause: error });
        }
```